packages/dialtone/lib/build/less/components/radio-checkbox.less

```diff
@@ -145,6 +145,16 @@
     cursor: pointer;
 }
 
+.d-checkbox__description {
+    display: flex;
+    box-sizing: border-box;
+    color: var(--dt-color-foreground-tertiary);
+    font-size: var(--dt-font-size-100);
+    font-family: inherit;
+    line-height: var(--dt-font-line-height-400);
+    fill: currentColor;
+  }
```

Comment:
This come from `d-description`?
The only think I can see here is this is `radio-checkbox.less` component, and we are removing it from `checkbox/checkbox.vue` if it is correct, cool

Reply:
hmm `d-description` = 

```
display: flex;
  box-sizing: border-box;
  margin-bottom: var(--su6);
  color: var(--black-600);
  font-size: var(--fs-100);
  font-family: inherit;
  line-height: var(--lh4);
  fill: currentColor;
```

Seems like we are missing  `margin-bottom: var(--su6);`, Im guessing this was intended, what was the reason?

Reply:
Yes, it comes from `d-description`

Reply:
this is ok